sharpy/structure/models/beam.py

```diff
@@ -544,7 +476,8 @@ def get_body(self, ibody):
         int_list_nodes = np.arange(0, ibody_beam.num_node, 1)
         for ielem in range(ibody_beam.num_elem):
             for inode_in_elem in range(ibody_beam.num_node_elem):
-                ibody_beam.connectivities[ielem, inode_in_elem] = int_list_nodes[ibody_nodes == ibody_beam.connectivities[ielem, inode_in_elem]]
+                ibody_beam.connectivities[ielem, inode_in_elem] = int_list_nodes[
```

Comment:
Another fix for converting arrays to scalars which creates a warning during unit test.

Reply:
good spot and fix!